MvvmCross/Platforms/Android/Presenters/MvxAndroidViewPresenter.cs

```diff
@@ -344,7 +344,7 @@ protected virtual void ShowHostActivity(MvxFragmentPresentationAttribute attribu
                 throw new NullReferenceException($"Fragment host not found when trying to show View {view.Name} as Nested Fragment");
 
             if (!fragmentHost.IsVisible)
-                throw new InvalidOperationException($"Fragment host is not visible when trying to show View {view.Name} as Nested Fragment");
+                MvxLog.Instance.Warn("Fragment host is not visible when trying to show View {0} as Nested Fragment", view.Name);
```

Comment:
Should probably be `MvxAndroidLog`

Reply:
As `MvxAndroidViewPresenter` is part of the main MvvmCross project which has access to the default static `MvxLog` internal instance. The `MvxAppCompatViewPresenter` on the other hand sits in a different .dll which makes use of the `MvxAndroidLog` IoC resolved internal instance.

Reply:
You are right 👍